src/middleware/index.js

```diff
@@ -31,7 +31,9 @@ middleware.regexes = {
 	timestampedUpload: /^\d+-.+$/,
 };
 
-middleware.applyCSRF = csrf();
+middleware.applyCSRF = csrf({
+	cookie: true,
+});
```

Comment:
These lines save the csrf token in a cookie called `_csrf` instead of inside `req.session`. No side-effects were found in this change.